source/_integrations/tessie.markdown

```diff
@@ -24,6 +26,32 @@ You must have a [Tessie](https://my.tessie.com/) account and [access token](http
 
 ## Entities
 
+### Binary Sensor
+
+The integration will create binary sensor entities for a variety of metrics that relate to your vehicles: 
+
+#### Charge State
+- Battery Heater
+- Battery Charing
+- Preconditioning Enabled
+- Scheduled Charging Enabled
+- Trip Charging Enabled
+
+#### Climate State
+- Auto Seat Climate Left
+- Auto Seat Climate Right
+- Auto Steering Wheel Climate
+- Overheat Protection Enabled
+- Overheat Protection Running
+
+#### Vehicle State
+- Dashcam Recording
+- User Present
+- Tyre Preasure Warning Front Left
+- Tyre Preasure Warning Front Right
+- Tyre Preasure Warning Rear Left
+- Tyre Preasure Warning Rear Right
```

Comment:
```suggestion
- Tyre Pressure Warning Front Left
- Tyre Pressure Warning Front Right
- Tyre Pressure Warning Rear Left
- Tyre Pressure Warning Rear Right
```

I think the standard is to use American English as well so maybe this should change to `Tire`

Reply:
I'll update the strings.json to reflect `Tire`